### ISSUE TITLE
KeyError: <whatever repo the queried package is to be found in>

### ISSUE THREAD

**Issue by Callisos** [openSUSE/opi#169]
When searching for Mesa-dri pkg w/ opi, I get the following error:
```
> LANG=c opi Mesa-dri
Searching repos for: Mesa-dri
 1. Mesa-dri
 2. Mesa-dri-devel
 3. Mesa-dri-32bit
 4. Mesa-dri-nouveau
 5. Mesa-dri-debuginfo
 6. Mesa-dri-nouveau-32bit
 7. Mesa-drivers-debugsource
 8. Mesa-dri-32bit-debuginfo
 9. Mesa-dri-nouveau-debuginfo
10. Mesa-dri-nouveau-32bit-debuginfo
Pick a number (0 to quit): 1
You have selected package name: Mesa-dri
Traceback (most recent call last):
  File "/usr/bin/opi", line 146, in <module>
    repo_query(args.query)
  File "/usr/bin/opi", line 85, in repo_query
    packages = opi.search_local_repos(selected_pkg_name) + packages
  File "/usr/lib/python3.6/site-packages/opi/__init__.py", line 214, in search_local_repos
    installable['repository'] = repos_by_name[repo_name]
KeyError: 'packman-essentials'
```
Note I've set a higher priority for the packman-essentials repo than the official one has by default where the Mesa-dri pkg is otherwise available in. So, when disabling the packman-essentials repo, I get the following error for a change:
```
> LANG=c opi Mesa-dri
Searching repos for: Mesa-dri
 1. Mesa-dri
 2. Mesa-dri-devel
 3. Mesa-dri-32bit
 4. Mesa-dri-nouveau
 5. Mesa-dri-debuginfo
 6. Mesa-dri-nouveau-32bit
 7. Mesa-drivers-debugsource
 8. Mesa-dri-32bit-debuginfo
 9. Mesa-dri-nouveau-debuginfo
10. Mesa-dri-nouveau-32bit-debuginfo
Pick a number (0 to quit): 1
You have selected package name: Mesa-dri
Traceback (most recent call last):
  File "/usr/bin/opi", line 146, in <module>
    repo_query(args.query)
  File "/usr/bin/opi", line 85, in repo_query
    packages = opi.search_local_repos(selected_pkg_name) + packages
  File "/usr/lib/python3.6/site-packages/opi/__init__.py", line 214, in search_local_repos
    installable['repository'] = repos_by_name[repo_name]
KeyError: 'repo-oss'
```

When disabling the repo-oss, opi starts to work OK for this specific queried pkg. Similarly, when searching for the wine pkg, I get the `KeyError: 'wine'` line at the end of the opi output and so on.

OS & opi versions:
```
> lsb-release -d
Description:    openSUSE Leap 15.5
```
```
> LANG=c zypper se -is opi
Loading repository data...
Reading installed packages...

S  | Name | Type    | Version           | Arch   | Repository
---+------+---------+-------------------+--------+----------------------
i+ | opi  | package | 4.2.0-bp155.2.9.1 | noarch | repo-backports-update
```

**Comment by asdil12** [comment]
Could you send me the output of this command?

```
(cd /etc/zypp/repos.d; \ls | while read line ; do echo -e "\n----\n$line:"; cat "$line" ; done)
```

**Comment by asdil12** [comment]
The fix will be in opi 4.4.0 - until then you can workaround by setting `showAlias` to `false`.
